modules/assim.sequential/R/sda.enkf.R

```diff
@@ -447,6 +459,15 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
       new.params[[i]] <- X_tmp[[i]]$params
     }
     
+    ## Trying to find a way to flag incomplete runs and drop them.
```

Comment:
I think a check like this is helpful  but need to think carefully about this:
1) If certain parameters, drivers, or IC in the ensemble leads to a crash, then their contribution to the forecast violates "missing at random". Does that matter in this context?
2) How does the restart code respond to missing ensemble members? Does it just try to skip them? 
3) If so, does that lead (over many cycles) to the progressive depauperization of the ensemble size. Could we replace lost members using a PF like resampling? Would that be valid?

Reply:
I just took this out so that we could get the pr through and I can come back to it later?

Reply:
yeah, coming back to this latter is fine